README.md

```diff
@@ -96,11 +96,19 @@ Utilize an Agile project management process to hone skill and comfort with build
 * Ruby 2.7.4
 * Rails 5.2.8
 
```

Comment:
This branch has a few changes to readme, but just a reminder we'll need to add the endpoints!

Reply:
For future reference, here are our current endpoints that need documenting:

GET `/api/v1/campsites/<CAMPSITE ID>`
GET `/api/v1/campsites?state_code=<STATE CODE>`
GET `/api/v1/campsites?q=<CAMPSITE NAME>`
GET `/api/v1/campsites?park_name=<PARK NAME>`

GET `/api/v1/favorites?user_id=1`
POST `/api/v1/favorites?user_id=1`
DELETE `/api/v1/favorites/<FAVORITE_ID>`